### PR TITLE
bmx7: sync with master branch

### PR DESCRIPTION
**Pull request by dangowrt**
Cherry pick commits from master branch
f0b852b bmx7: add config trigger
ab3529a bmx7: Avoid namespace collision with libubox.

### CODE REVIEW

**Comment by dangowrt** [comment]
@cotequeiroz @aparcar @mwarning

**Comment by aparcar** [comment]
ack

File: bmx7/Makefile [hunk 1]
```diff
@@ -33,7 +33,7 @@ PKG_SOURCE_URL:=git://github.com/bmx-routing/bmx7.git
 PKG_REV:=9883383dc26df16da67b9ef7ba99efe62f79c4e7
 PKG_MIRROR_HASH:=012178aba42016d3e0961715f99bf1e322c2c6c22b1cc6635375468360cb4d68
 PKG_VERSION:=r2018042501
-PKG_RELEASE:=1
```
Comment (mwarning) [inline bmx7/Makefile]: Why jump from 1 to 3? But it is no problem.

**Comment by mwarning** [comment]
@axn yay or nay?

**Comment by mwarning** [comment]
Looks like enough yay for me.

**Pull request merged**